### PR TITLE
Rewrite README based on image spec

### LINKED ISSUE #200: Friendly README
The readme should have some TODO sections filled out. Maybe look similar in form to https://github.com/opencontainers/image-spec/blob/master/README.md

### PR DESCRIPTION
**Pull request by jdolitsky**
Designed to look similar to https://github.com/opencontainers/image-spec/blob/master/README.md

Fixes #200

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,41 +1,159 @@
...
+which fully adheres to the OCI Image Format. The Runtime Spec is relevant to the Distribution Spec in that they both support OCI Images,
+and that container runtimes use the APIs defined in the Distribution Spec to fetch pre-built container images and run them.
+
+The Distribution Spec is also designed generically enough to be leveraged as a distribution mechanism for
+any type of content. The format of uploaded manifests, for example, need not necessarily adhere to the OCI Image Format
+so long as it references the blobs which comprise a given artifact.
+
+The [OCI Artifacts project](https://github.com/opencontainers/artifacts) is an effort to provide guidance on how to
+properly define and distribute content using the Distribution Spec for artifacts which are not container filesystem bundles,
+in a way that is mostly compatible with the existing schemas defined in the Image Format Spec.
```
Comment (jdolitsky) [inline README.md:39]: This is all new content. Tried to give the bigger picture

File: README.md [hunk 2]
```diff
@@ -1,41 +1,159 @@
...
+which fully adheres to the OCI Image Format. The Runtime Spec is relevant to the Distribution Spec in that they both support OCI Images,
+and that container runtimes use the APIs defined in the Distribution Spec to fetch pre-built container images and run them.
+
+The Distribution Spec is also designed generically enough to be leveraged as a distribution mechanism for
+any type of content. The format of uploaded manifests, for example, need not necessarily adhere to the OCI Image Format
+so long as it references the blobs which comprise a given artifact.
+
+The OCI Artifacts project is an effort to provide guidance on how to
+properly define and distribute content using the Distribution Spec for artifacts which are not container filesystem bundles,
+in a way that is mostly compatible with the existing schemas defined in the Image Format Spec.
```
Comment (jdolitsky) [inline README.md:39]: This is all new content. Tried to give the bigger picture

File: README.md [hunk 3]
```diff
@@ -1,41 +1,159 @@
...
```
Comment (jdolitsky) [inline README.md:39]: This is all new content. Tried to give the bigger picture

Comment (jzelinskie) [reply README.md:39]: For each paragraph in this section, I'd make the the specification being reference links, too.

```md
The [OCI Image Format Specification] ...

The [OCI Runtime Specification] ...

The [OCI Distribution Spec] ...

The [OCI Artifacts] ...

[OCI Image Format Specification]: https://github.com/opencontainers/image-spec
[OCI Image Runtime Specification]: https://github.com/opencontainers/runtime-spec
[OCI Image Distribution Specification]: https://github.com/opencontainers/distribution-spec
[OCI Image Artifacts Specification]: https://github.com/opencontainers/artifacts

```

Comment (jzelinskie) [reply README.md:39]: Also notice that I tried to be consistent with usage of "OCI" when formally naming things.

**Comment by jzelinskie** [comment]
Looks really good to me, only a basic nitpick on the net-new content.

**Comment by jdolitsky** [comment]
@jzelinskie ty for review, back at ya

**Comment by jdolitsky** [comment]
CI passing now, ping @opencontainers/distribution-spec-maintainers

**Review by vbatts — APPROVED** [review]
:+1:

**Comment by jzelinskie** [comment]
LGTM

**Pull request merged**